components/combobox/index.css

```diff
@@ -0,0 +1,451 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
```

Comment:
lets make this 2023

Reply:
Updated in [908252e](https://github.com/adobe/spectrum-css/pull/1556/commits/908252e02321c7a8fc002e709fcad8fdc6350de6)